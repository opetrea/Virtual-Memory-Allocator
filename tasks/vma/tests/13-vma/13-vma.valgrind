==91873== Memcheck, a memory error detector
==91873== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91873== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91873== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91873== Parent PID: 91872
==91873== 
==91873== 
==91873== HEAP SUMMARY:
==91873==     in use at exit: 0 bytes in 0 blocks
==91873==   total heap usage: 11 allocs, 11 frees, 8,416 bytes allocated
==91873== 
==91873== All heap blocks were freed -- no leaks are possible
==91873== 
==91873== For lists of detected and suppressed errors, rerun with: -s
==91873== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
