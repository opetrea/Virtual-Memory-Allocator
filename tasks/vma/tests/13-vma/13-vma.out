This zone was already allocated.
Total memory: 0x14 bytes
Free memory: 0x12 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x5 - 0x7
Miniblock 1:		0x5		-		0x7		| RW-
Block 1 end
